components/hab/src/main.rs

```diff
@@ -242,6 +242,13 @@ async fn start(ui: &mut UI, feature_flags: FeatureFlag) -> Result<()> {
                         _ => unreachable!(),
                     }
                 }
+                ("rbac", Some(m)) => {
+                    match m.subcommand() {
+                        ("set", Some(sc)) => sub_origin_member_role_set(ui, sc).await?,
+                        ("show", Some(sc)) => sub_origin_member_role_show(ui, sc).await?,
+                        _ => unreachable!(),
+                    }
+                }
```

Comment:
Instead of parsing this here using `clap` methods, could we move it to part of this [match statement](https://github.com/habitat-sh/habitat/blob/9472ea6f36ff1d2aaba6be16a736a60ab8d4ad1e/components/hab/src/main.rs#L150). This would allow us to use the `Rbac` type instead of `get_matches`.